documentation/modules/configuring/ref-list-of-kafka-cluster-resources.adoc

```diff
@@ -33,20 +33,18 @@ The following resources are created by the Cluster Operator in the Kubernetes cl
 `_cluster-name_-kafka`:: StatefulSet which is in charge of managing the Kafka broker pods.
 `_cluster-name_-kafka-_idx_`:: Pods created by the Kafka StatefulSet.
 `_cluster-name_-kafka-brokers`:: Service needed to have DNS resolve the Kafka broker pods IP addresses directly.
-`_cluster-name_-kafka-bootstrap`:: Service can be used as bootstrap servers for Kafka clients.
-`_cluster-name_-kafka-external-bootstrap`:: Bootstrap service for clients connecting from outside the Kubernetes cluster. This resource is created only when an external listener is enabled.
-`_cluster-name_-kafka-_pod-id_`:: Service used to route traffic from outside the Kubernetes cluster to individual pods. This resource is created only when an external listener is enabled.
-`_cluster-name_-kafka-external-bootstrap`:: Bootstrap route for clients connecting from outside the Kubernetes cluster. This resource is created only when an external listener is enabled and set to type `route`.
-`_cluster-name_-kafka-_pod-id_`:: Route for traffic from outside the Kubernetes cluster to individual pods. This resource is created only when an external listener is enabled and set to type `route`.
+`_cluster-name_-kafka-bootstrap`:: Service can be used as bootstrap servers for Kafka clients connecting from within the Kubernetes cluster.
+`_cluster-name_-kafka-_listener-name_-bootstrap`:: Bootstrap service for clients connecting from outside the Kubernetes cluster. This resource is created only when an external listener is enabled.
+`_cluster-name_-kafka-_listener-name_-_pod-id_`:: Service used to route traffic from outside the Kubernetes cluster to individual pods. This resource is created only when an external listener is enabled.
```

Comment:
I do not think this is completely correct. The old service names will be used for backwards compatibility when the listener name is `external` and port is `9094`. These new services will be used for all other external listeners. So maybe we want to have mentioned both and have it explained what is used when?